clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForE4E.ts

```
@@ -356,4 +351,33 @@ export class CopybookDownloaderForE4E {
       this.storagePath,
     );
   }
+  public async getProfileInfo(profile: string = "") {
+    const partialProfile = asPartialProfile(profile);
+    if (this.E4EProfiles.has(profile)) {
+      return this.E4EProfiles.get(profile);
+    }
+    const resolvedProfile = await this.e4e.getProfileInfo(partialProfile);
+    if (!(resolvedProfile instanceof Error)) {
+      this.E4EProfiles.set(profile, resolvedProfile);
+      return this.E4EProfiles.get(profile);
```

```suggestion
      return resolvedProfile;
```

---

And if you get an error? Do we just silently ignore it?